helpers/istio.go

```diff
@@ -35,7 +35,9 @@ spec:
       protocol: HTTP
     hosts:
     - "{{ .Host }}"
-
+`
+	if serviceHost != "" {
+		templateSource += `
```

Comment:
any reason to not include this on the template?

Reply:
Sorry, I do not understand the question...

Reply:
I see you have a template and depending on the `serviceHost` value you concatenate it with another template. Why not have only one template that checks the value of `serviceHost` and include the "second template" conditionally? for example:

```
{{ if serviceHost != "" }}
second template content...
{{ end }}
```

IMO it makes it easier to read the template logic.

Reply:
ah, good point. I did not try to find out how to do it inside template so I did it the traditional way.
But I agree your solution is better